### ISSUE TITLE
accessibility zoom overwidth

### ISSUE THREAD

**Issue by openmindculture** [openmindculture/ingo-steinke.de#319]
With text size increased in Android system settings, form fields overflow viewport width in mobile Firefox, reproducible with text-only zoom in Firefox on desktop (activate in classic top menu invoked with Alt key, View -> Zoom -> check zoom text only, them zoom in using the same menu).

The overflow violates WCAG 2.x Success Criterion 1.4.4: Resize Text (Level AA).

<img width="580" height="1290" alt="Image" src="https://github.com/user-attachments/assets/3f28d267-4c31-4ea9-8fca-08e8e7ef79d6" />

**Comment by openmindculture** [comment]
Fix:
- unset implicit `min-width` on form field and container (`fieldset`)
- set `fieldset max-width`

```css
.contactform fieldset {
 min-width: 0;
 max-width: 100vw;
}
.contactform-row input
min-width: 0;
```
